puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -31,13 +32,14 @@
   }
 
   exec { "jenkins-jobs-update-${config_name}":
-    command => "jenkins-jobs --conf ${inifile} update ${directory}/${config_name} > /var/cache/jjb.xml",
+    command => "jenkins-jobs --conf ${inifile} update ${directory}/${config_name} foreman-infra-jenkins-job-update",
```

Comment:
:-1: please revert. This is a defined type and should be generic. If anything, it should be a parameter.

Reply:
I missed that, gets confusing for me which things are generic and which things are specific for our infra. https://github.com/theforeman/foreman-infra/pull/1556